### PR TITLE
commas in chunk names

### PR DESCRIPTION
**Pull request by mpadge**
Only really matters for things like my [precommit check for unique chunk names throughout](https://github.com/ropenscilabs/statistical-software-peer-review/commit/3c98733db2863affb0877cc8459d8db978a5b891).

### CODE REVIEW

**Comment by maelle** [comment]
thank you!! 

these things are hard to parse... https://github.com/lockedata/namer

**Comment by mpadge** [comment]
oh yeah, i'd forgotten about `namer` :smile:

**Comment by maelle** [comment]
well it's not really useful if you want _informative_ chunk names :grin:

**Comment by mpadge** [comment]
Plus taking care of it all with precommit hooks is way easier because it's automatic and simply tells you to insert your own informative one if you miss any
